api_tests/base/test_views.py

```diff
@@ -38,6 +38,8 @@
 for mod in URLS_MODULES:
     urlpatterns = mod.urlpatterns
     for patt in urlpatterns:
+        if hasattr(patt, 'name') and patt.name == 'raw-metrics-view':
```

Comment:
Could you describe for me why this block?

Reply:
Since this is a development only endpoint, I did not find it necessary to include 'embed' functionality. This block excludes the raw metrics view from the embed check.